alerts.go

```diff
@@ -17,94 +17,100 @@ type AlertsService interface {
 var _ AlertsService = (*Client)(nil)
 
 type Alert struct {
-	ID            string  `json:"id"`
-	AppID         string  `json:"app_id"`
-	ContainerType string  `json:"container_type"`
-	Metric        string  `json:"metric"`
-	Limit         float64 `json:"limit"`
-	RemindEvery   string  `json:"remind_every"`
-	Disabled      bool    `json:"disabled"`
-	SendWhenBelow bool    `json:"send_when_below"`
+	ID                    string        `json:"id"`
+	AppID                 string        `json:"app_id"`
+	ContainerType         string        `json:"container_type"`
+	Metric                string        `json:"metric"`
+	Limit                 float64       `json:"limit"`
+	Disabled              bool          `json:"disabled"`
+	SendWhenBelow         bool          `json:"send_when_below"`
+	DurationBeforeTrigger time.Duration `json:"duration_before_trigger"`
+	RemindEvery           string        `json:"remind_every"`
 }
 
 type AlertsRes struct {
 	Alerts []*Alert `json:"alerts"`
 }
 
 type AlertRes struct {
-	Alert Alert `json:"alert"`
+	Alert *Alert `json:"alert"`
```

Comment:
Why a pointer?

Reply:
All methods of the `AlertsService` interface return a `*Alert`. So it's kind of straightforward for me to have a `*Alert` here.

Reply:
Ok, I'll open a issue saying not to use pointer then :D